kombu/utils/functional.py

```diff
@@ -339,17 +340,17 @@ def retry_over_time(fun, catch, args=None, kwargs=None, errback=None,
                              interval_max + interval_start,
                              interval_step, repeatlast=True)
     end = time() + timeout if timeout else None
-    for retries in count():
+    for retry_attempt in count():
```

Comment:
is this renaming must be needed?

Reply:
I felt retries is not a clear name -- it requires more code inference to see what is happening when reading through things quickly.

Reply:
Well I think both names are not ideal because the first attempt is not a retry -- its the initial request.   Maybe it should be `for request_attempt` for code clarity.